src/manifest/pfm.rs

```diff
@@ -897,4 +890,65 @@ mod test {
         assert_eq!(imgs[1].region(1), Some(Region::new(0x28000, 0x1000)));
         assert!(imgs[1].region(2).is_none());
     }
+
+    #[test]
+    fn baked_pfm1() {
+        let sha = ring::sha256::Builder::new();
+        let (mut rsa, mut signer) = test_rsa();
+        let mut arena = vec![0; 1024];
+        let arena = BumpArena::new(&mut arena);
+
+        let bytes = Ram(testdata::PFM_RSA1);
+        // NOTE: Manticore and MS Cerberus currently have inconsistent signing
+        // schemes, so we disable signature verification (for now).
+        /*let container = Container::parse_and_verify(
+            &bytes,
+            &sha,
+            &mut rsa,
+            &arena,
+            &OutOfMemory,
+        )
+        .unwrap();*/
+        let container = Container::parse(&bytes, &arena).unwrap();
```

Comment:
A question about idiomatic rust: According to [the docs](https://doc.rust-lang.org/rust-by-example/testing/unit_testing.html#tests-and-), tests are permitted to return `Result<()>` so you can used `?` in place of `unwrap()`.

Do we allow that in our codebase?

Reply:
So, I believe that's a modern style; I've never done that but that's because I've been writing this language for a while, so I missed when this was added.

I think this is a good idea. Do you think we should file an issue to update the tests? I think it's a good task to leave to someone less-familiar with the codebase so they can get a feel for what all the names of everything is.

Reply:
SGTM.  Do we also have rust styleguide which would inform about such things?

https://github.com/lowRISC/opentitan/issues/5844